src/rngs/thread.rs

```diff
@@ -112,7 +112,7 @@ impl RngCore for ThreadRng {
         unsafe { (*self.rng.get()).next_u64() }
     }
 
-    fn fill_bytes(&mut self, bytes: &mut [u8]) {
+    fn fill_bytes(&mut self, dest: &mut [u8]) {
         unsafe { (*self.rng.get()).fill_bytes(bytes) }
```

Comment:
One more line to change...

Reply:
Sorry, I ran tests on Master because apparently today is Monday. :)